lib/openshift/cluster_operator.rb

```diff
@@ -11,5 +11,28 @@ def version_exists?(version:, user: nil, cached: true, quiet: false)
       res = self.versions.select { |v| v == version }
       res.count != 0
     end
+    # @return an array of relatedObjects
+    #   relatedObjects:
+    #   - group: ""
+    #     name: openshift-cluster-storage-operator
+    #     resource: namespaces
+    #   - group: config.openshift.io
+    #     name: cluster
+    #     resource: infrastructures
+    #   - group: storage.k8s.io
+    #     name: gp2
+    #     resource: storageclasses
+    def related_objects(user: nil, cached: true, quiet: false)
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      rr.dig('status', 'relatedObjects')
+    end
```

Comment:
Could you make this return objects from the OpenShift classes instead of the raw map?

See my comment:
https://github.com/openshift/cucushift/pull/7026#issuecomment-509797154

Reply:
@akostadinov I've added a related_object class under `lib/openshift/flakes` to represent the hash.  PTAL again.  Thanks!

Reply:
Actually the related objects are not a special kind of object but:

Resource name "namespaces" of kind "openshift-cluster-storage-operator",
resource name "cluster" of kind "config.openshift.io"
etc.

So we need a way to generate these objects when they are already defined, or a fallback object otherwise. If I can't explain clear I can take a look later to do myself or at least pseudo code to make more clear. Please let me know.

Reply:
bump, what to discuss this?